src/app/dashboard-widgets/add-codebase-widget/add-codebase-widget.component.html

```diff
@@ -3,14 +3,18 @@
     <div class="card-pf-heading f8-card-heading">
       <f8-feature-toggle featureName="AppLauncher">
         <div class="card-pf-heading-details f8-card-heading-details" default-level>
-          <a id="spacehome-codebases-add-space-button" class="btn btn-link f8-card-heading-btn-link" (click)="addToSpace.emit()">
-            <i class="pficon pficon-add-circle-o"></i>
-          </a>
-        </div>
-        <div class="card-pf-heading-details f8-card-heading-details" user-level>
+          <ng-container *ngIf="userOwnsSpace">
             <a class="btn btn-link f8-card-heading-btn-link" (click)="addToSpace.emit()">
-                <i class="pficon pficon-add-circle-o"></i>
+              <i id="default-level-add-to-space-btn" class="pficon pficon-add-circle-o"></i>
             </a>
+          </ng-container>
+          <div class="card-pf-heading-details f8-card-heading-details" user-level>
+            <ng-container *ngIf="userOwnsSpace">
+              <a class="btn btn-link f8-card-heading-btn-link" (click)="addToSpace.emit()">
```

Comment:
Other than the ID, this button is exactly the same as line 7. I see that the AppLauncher feature has been changed, which makes me question if this should be removed?